jetty-util/src/main/java/org/eclipse/jetty/util/component/Dumpable.java

```diff
@@ -181,14 +181,18 @@ else if (object instanceof Map)
                 dumpObjects(out, nextIndent, item);
         }
     }
-
+    
     static void dumpContainer(Appendable out, String indent, Container object, boolean last) throws IOException
     {
         Container container = object;
         ContainerLifeCycle containerLifeCycle = container instanceof ContainerLifeCycle ? (ContainerLifeCycle)container : null;
         for (Iterator<Object> i = container.getBeans().iterator(); i.hasNext(); )
         {
             Object bean = i.next();
+
+            if (container instanceof DumpableContainer && !((DumpableContainer)container).isDumpable(bean))
```

Comment:
OK, This is a actually a much better way to accomplish this.

Reply:
Can `dumpContainer` be moved to be a static method of `DumpableContainer` to keep those concepts together?